.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,119 @@
+name: " Bug Report"
+description: I would like to report a bug.
+title: "[ BUG] <write a small description here>"
```

Comment:
The title format has inconsistent spacing `"[ BUG]"`. For consistency, it should be `"[BUG]"` without extra spaces.